includes/User.php

```diff
@@ -246,6 +247,11 @@ class User {
 	static $idCacheByName = array();
 
 	/**
+		* @var string the service auth token (currently helios); should NEVER be cached
+	 */
+	private $globalAuthToken = null;
```

Comment:
I think that any class variables end up as part of the Memcache object. Will that break anything with multiple sessions?